capture-sdk/screen-api-example-app/src/main/res/values/styles.xml

```diff
@@ -6,6 +6,7 @@
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="android:tint">@color/white</item>
```

Comment:
[Request] Android styles are black magic and changing the tint color for the whole theme can change lots of unintented things.

I recommend limiting the tint color change to the theme of the action bar:
```suggestion
        <item name="actionBarTheme">@style/AppActionBar</item>
    </style>

    <style name="AppActionBar" parent="ThemeOverlay.MaterialComponents.Dark.ActionBar">
        <item name="android:tint">@color/white</item>
    </style>
```

Reply:
Done.

At first I did the same, but I guessed this just affects our example app and it is not important. You are absolutely right. I should consider the right job for every use case. 💯